Packet++/src/Packet.cpp

```diff
@@ -31,8 +31,8 @@ Packet::Packet(size_t maxPacketLen) :
 {
 	timeval time;
 	gettimeofday(&time, NULL);
-	uint8_t* data = new uint8_t[m_MaxPacketLen];
-	memset(data, 0, m_MaxPacketLen);
+	uint8_t* data = new uint8_t[maxPacketLen];
+	memset(data, 0, maxPacketLen);

```

Comment:
why does this change matter?

Reply:
This change gives more chances to compiler to produce effective binary code. Let's get the part of current constructor's body:
```
Packet::Packet(size_t maxPacketLen) :
...
  m_MaxPacketLen(maxPacketLen),
...
{
  ...
  uint8_t* data = new uint8_t[m_MaxPacketLen];
  memset(data, 0, m_MaxPacketLen);
}
```
In this case the compiler have to read the value of `m_MaxPacketLen` before allocating the memory and calling the `memset`. The new approach in coding avoids the need to read already known value that is stored in `maxPacketLen`.
```
Packet::Packet(size_t maxPacketLen) :
...
  m_MaxPacketLen(maxPacketLen),
...
{
  ...
  uint8_t* data = new uint8_t[maxPacketLen];
  memset(data, 0, maxPacketLen);
}
```

Reply:
this is a nice optimization :)